src/main/java/frc/robot/subsystems/LiftSubsystem.java

```diff
@@ -123,6 +131,22 @@ public void initDefaultCommand() {
         setDefaultCommand(moveLift());
     }
 
+    public Command foldArms() {
```

Comment:
I'm not a fan of this name, can we do something like `lockArms`?

Reply:
> I'm not a fan of this name, can we do something like `lockArms`?

Changed to lockArms